permissions/plugin-icedq.yml

```diff
@@ -0,0 +1,10 @@
+---
+name: "icedq"
+github: "jenkinsci/icedq-plugin"
+paths:
+- "com/icedq/ci/plugin/icedq"
+developers:
+- "devlopertoranainc"
+- "toranainc"
```

Comment:
https://issues.jenkins-ci.org/secure/ViewProfile.jspa?name=toranainc 404. This user is wrong.